packages/react/src/analytics/integrations/Vitorino/providers/Riskified/Riskified.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  EventData,
+  IntegrationFactory,
+  integrations,
+  LoadIntegrationEventData,
+  StrippedDownAnalytics,
+  TrackTypes,
+  TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+import get from 'lodash/get';
+import type { RiskifiedIntegrationOptions } from './types';
+
+class Riskified extends integrations.Integration<RiskifiedIntegrationOptions> {
+  private isReady: boolean;
+  private siteId: string;
+  private baseUrl: string;
+
+  /**
+   * Creates an instance of Riskified and sets the script on the page.
+   *
+   * @param options - Custom options for the integration.
+   * @param loadData - Analytics's load event data.
+   */
+  constructor(
+    options: RiskifiedIntegrationOptions,
+    loadData: LoadIntegrationEventData,
+    analytics: StrippedDownAnalytics,
+  ) {
+    super(options, loadData, analytics);
+
+    this.isReady = false;
+    this.siteId = options.siteId || 'farfetch.com';
+    this.baseUrl = `https://beacon.riskified.com?shop=${this.siteId}&sid=`;
```

Comment:
I am not sure if this should be set on the source code due to security concerns. I would have this option being passed by the consumer in `analytics.addIntegration` and have documentation that explains that this is the value that should be set.

Reply:
This is done already on Vitorino script. The objective of this initiative it's to deprecate the usage of Vitorino script, maintaining behavior but from blackout side. Besides that, yes, these keys could be removed from integration and set only by addIntegration, but it needs to be discussed because it doesn't be aligned with task requirements.

Reply:
Update: It was removed the defaults, as we discuss offline.